shopinvader_schema_invoice/schemas/amount.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+from odoo.tools.float_utils import float_round
+
+
+class InvoiceAmount(StrictExtendableBaseModel, extra="ignore"):
+    amount_tax: float | None
+    amount_untaxed: float | None
+    amount_total: float | None
+    amount_due: float | None
+
+    @classmethod
+    def from_one(cls, odoo_rec):
```

Comment:
from_one what? IMO it's important to be explicit on the expected model. Is-it from a sale order or a product or ????

Reply:
ehh I was trying to find a generic naming to apply to all models so that we don't have to remember every time which name the dev chosen when coding.
What about `from_record` and `from_records`?
I mean, the purpose of a class is pretty clear. Is called "Invoice". For the main record where it comes from I wouldn't elaborate that much.

Reply:
My preference is `from_invoice`. Or `from_account_move`.